app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,67 @@
+class AlbumsController < ApplicationController
+  def index
+    @albums = Album.order(ranking: :desc)
```

Comment:
You could also do the same but writing a scope for this method in Album model. 
For instance, in you album model 
scope :desc_order, -> { order('ranking DESC') }
and then index would look: 
@albums = Album.desc_order